src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/MethodologySummaryPage.tsx

```diff
@@ -11,18 +11,60 @@ import SummaryListItem from '@common/components/SummaryListItem';
 import Tag from '@common/components/Tag';
 import WarningMessage from '@common/components/WarningMessage';
 import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import {
+  ReleaseRouteParams,
+  releaseSummaryRoute,
+} from '@admin/routes/releaseRoutes';
 import React from 'react';
 import { generatePath, RouteComponentProps } from 'react-router';
+import { Link } from 'react-router-dom';
 
 const MethodologySummaryPage = ({
   match,
 }: RouteComponentProps<MethodologyRouteParams>) => {
   const { methodologyId } = match.params;
 
-  const { value: currentMethodology, isLoading } = useAsyncHandledRetry(() =>
+  const { value: realMethodology, isLoading } = useAsyncHandledRetry(() =>
     methodologyService.getMethodology(methodologyId),
   );
 
+  const myFakeMethodology = {
```

Comment:
Could hoist this variable out of the component scope to save having to build this object on every render